packages/edit-site/src/components/preferences-modal/index.js

```diff
@@ -86,6 +86,13 @@ export default function EditSitePreferencesModal() {
 						) }
 						label={ __( 'Display block breadcrumbs' ) }
 					/>
+					<EnableFeature
+						featureName="allowRightClickOverrides"
+						help={ __(
+							'Allows contextual menus via right-click, overriding browser defaults.'
+						) }
```

Comment:
Do we need to communicate that it's specific to the List view here? I don't think it has effect anywhere else in the editor (?)

Reply:
Oh, for the user facing help text, that's a great idea! Perhaps `Allows contextual list view menus via right-click...` or something like that.